projects/LSKNet/configs/lsk_s_ema_fpn_1x_dota_le90.py

```diff
@@ -0,0 +1,154 @@
+_base_ = [
+    '../../../configs/_base_/datasets/dota_ms.py',
+    '../../../configs/_base_/schedules/schedule_1x.py',
+    '../../../configs/_base_/default_runtime.py'
```

Comment:
Here we can use `mmrotate::` to replace these dots

Example
```
'mmrotate::_base_/datasets/dota_ms.py'
```

Reply:
Hi @zcablii, this is a syntax sugar brought with OpenMMLab 2.0, which can easily which can easily inherit configuration files from different MM libraries.

https://github.com/open-mmlab/mmdetection/blob/dev-3.x/projects/EfficientDet/configs/efficientdet_effb0_bifpn_8xb16-crop512-300e_coco.py